src/dev/impl/DevToys/Helpers/CertificateHelper.cs

```diff
@@ -1,18 +1,18 @@
-﻿using System;
+﻿#nullable enable
+
+using System;
 using System.Linq;
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
 using System.Text;
 
 namespace DevToys.Helpers
 {
-#nullable enable
     internal static class CertificateHelper
     {
         private const string BeginCertificate = "BEGIN CERTIFICATE";
         private const string BeginCertificateRequest = "BEGIN CERTIFICATE REQUEST";
-        internal const string IncorrectPassword = "The specified network password is not correct";
-        internal const string FormatNotSupported = "Certificate Request format not currently supported.";
+        private const int IncorrectPassword = -2147024810;
```

Comment:
Can we please refactor it to something like

```
// "The specified password is not correct." (ERROR_INVALID_PASSWORD)
private const int ERROR_INVALID_PASSWORD_HRESULT = unchecked((int)0x80070056);
```

Reply:
Thank you, that indeed is the correct way to do this: https://github.com/microsoft/IIS.Administration/blob/f03620a9eb95f053fbe437f8aef84f89a6513a8b/src/Microsoft.IIS.Administration.WebServer.CentralCertificates/Errors.cs#L9